OAK-Marketplace-Emotion-Recognition/README.md

```diff
@@ -0,0 +1,58 @@
+This contains the code for **Emotion Recognition using OpenCV AI KIT and Modelplace**. For more information - visit [OpenCV.AI](https://opencv.ai/post/emotional-recognition).
+
+# OAK Emotion Recognition
+
+![](images/usage_demo.gif)
+
+## Requirements
+
+- Install Python 3.7 or higher and run: `python3 -m pip install -r requirements.txt`
+
+- Install [Emotion Recognition OAK model](https://github.com/opencv-ai/opencv-blog/tree/main/OAK-Marketplace-Emotion-Recognition#installation-of-emotion-recognition-oak-package)
+
+- Install [FFmpeg](https://ffmpeg.org/download.html)
+
+## Installation of Emotion Recognition OAK package:
+
+1. Clone the [repository](https://github.com/opencv-ai/oak-model-samples) with OAK model samples by the following command:
+
+   ```bash
+   git clone https://github.com/opencv-ai/oak-model-samples.git && \
+   cd oak-model-samples && \
+   git checkout 54fa4fe8bd2bb42d6bae49213dbbe6b2f1ce5c5e
+   ```
+
+2) Set up Emotion Recognition OAK package:
+
+   2.1 `cd emotion_recognition_retail`
+
+   2.2 `export PATH_TO_EMOTION_RECOGNITION_MODEL=$(pwd)`
+
+   2.3 `python3 setup.py bdist_wheel && rm -R build/ *.egg-info`
```

Comment:
We need to install the wheel package
`pip install wheel && python3 setup.py bdist_wheel && rm -R build/ *.egg-info`